jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -5076,7 +5076,13 @@ public String descriptorString() {
 		if (this.isArray()) {
 			return name;
 		}
```

Comment:
avoid multiple return statements and use `else if`.
```suggestion
		if (this.isArray()) {
		} else if (this.isPrimitiveClass()) {
```